cpp/tests/groupby/merge_sets_tests.cpp

```diff
@@ -45,9 +47,17 @@ auto merge_sets(vcol_views const& keys_cols, vcol_views const& values_cols)
   requests[0].aggregations.emplace_back(
     cudf::make_merge_sets_aggregation<cudf::groupby_aggregation>());
 
-  auto gb_obj = cudf::groupby::groupby(cudf::table_view({*keys}));
-  auto result = gb_obj.aggregate(requests);
-  return std::pair(std::move(result.first->release()[0]), std::move(result.second[0].results[0]));
+  auto const result     = cudf::groupby::groupby(cudf::table_view({*keys})).aggregate(requests);
+  auto const sort_order = cudf::sorted_order(result.first->view(), {}, {cudf::null_order::AFTER});
+  auto const sorted_vals =
+    std::move(cudf::gather(cudf::table_view{{result.second[0].results[0]->view()}}, *sort_order)
+                ->release()
+                .front());
+
+  auto result_keys = std::move(cudf::gather(result.first->view(), *sort_order)->release().front());
```

Comment:
Seems like this is the same as just calling `cudf::sort_by_key`?
```
auto result_keys_values = cudf::sort_by_key( cudf::table_view{result.first->view()}
                                             cudf::table_view{result.first->view(), result.second[0].results[0]->view()},
                                             {}, {cudf::null_order::AFTER});
auto result_keys = std::move(result_keys_values->column(0));
auto result_values = std::move(result_keys_values->column(1));
```

Reply:
Ah yes, it is `sort_by_key`. Thanks.